phpstan.neon

```diff
@@ -0,0 +1,14 @@
+includes:
+  - vendor/szepeviktor/phpstan-wordpress/extension.neon
+
+parameters:
+	# Level 9 is the highest level
+	level: max
```

Comment:
Oh this is 🔥

Reply:
I copied it from `create-wordpress-plugin` for consistency.